echemdb/data/cv/entry.py

```diff
@@ -100,40 +100,123 @@ def __getitem__(self, name):
         """
         return Descriptor(self.package.descriptor)[name]
 
-    def df(self, yunit=None):
+    def xy_units(self, xunit=None, yunit=None):
```

Comment:
Why don't split this into `x_unit` and `y_unit`? There seems to be no interaction between the two parts of this method.